css/cssom-view/elementFromPoint-display-contents.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>CSSOM View - 5 - extensions to the Document interface</title>
+    <link rel="author" title="Luke Warlow" href="mailto:lwarlow@igalia.com">
+    <link rel="help" href="http://www.w3.org/TR/cssom-view/#extensions-to-the-document-interface">
+    <meta name="flags" content="dom">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <style>
+        html,body {
+            padding: 0;
+            margin: 0;
+        }
+        #targetDiv {
+            display:contents;
+        }
+    </style>
+</head>
+<body>
+    <div id="targetDiv">Text Content</div>
+    <script>
+        var element = document.elementFromPoint(5, 5);
```

Comment:
So... Thinking a bit more about this, I think both returning `targetDiv` and `body` might be reasonable? Specially once you consider shadow DOM behavior / slots. You might want to get the element that a click event would be dispatched to, if you hit a text node...

If the spec is unclear, I suggest landing these with `.tentative.html` and pointing to a CSS spec issue?

I agree in any case that `document.elementsFromPoint(x, y).includes(document.elementFromPoint(x, y))` should be true. Maybe add a non-tentative test checking that?

Reply:
Well, the point is that the text node does have a box. So the question is whether you retarget the text node to the display: contents element or just skip it.

Reply:
Well isn't text in the text node actually a (text sequence)[https://www.w3.org/TR/css-display-4/#css-text-sequence]?